### DISCUSSION TITLE
[Вопрос] БГ с ботами

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#229] [Q&A]
Всем привет!
trickerer, возможно ли сделать бг со случайными ботами когда регаешь поле боя, например в пати со своими ботами?

**Comment by trickerer** [comment]
Невозможно. Боты не способны назначать себе точки перемещения, они привязываются к игроку, либо к начальной позиции, и никуда не путешествуют. Их самостоятельность в свободном состоянии ограничивается самозащитой, хотя иногда это может привести к массовой бойне, если в одной точке на расстоянии до 20 метров находится множество ботов враждебных фракций.

**Comment by xBarselonx** [comment]
Спасибо за ответ.
